src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -482,7 +522,9 @@ private static void handleFailedConfigs(
     }
 
     /**
-     * Generates a report for a single repository at {@code repoReportDirectory}.
+     * Generates a report for a single repository at {@code repoReportDirectory} based on {@code commitSummary}
+     * and {@code authorshipSummary}.
+     *
      * @return A list of paths to the JSON report files generated for this report.
      */
     private static List<Path> generateIndividualRepoReport(
```

Comment:
Should move the `repoReportDirectory` variable to line 530 and keep the rest in line 531.